Content.Server/Atmos/EntitySystems/HeatExchangerSystem.cs

```diff
@@ -21,6 +21,7 @@ public sealed class HeatExchangerSystem : EntitySystem
     [Dependency] private readonly AtmosphereSystem _atmosphereSystem = default!;
     [Dependency] private readonly IConfigurationManager _cfg = default!;
     [Dependency] private readonly NodeContainerSystem _nodeContainer = default!;
+    [Dependency] private IGameTiming _gameTiming = default!;
```

Comment:
_timing is standard

Reply:
atmos code uses _gameTiming

Reply:
it also uses EntityManager.X everywhere doesnt mean it good